monitor/remoteOperations.go

```diff
@@ -40,7 +40,7 @@ func GetAllRemote(esClient es.Client, destinationsMap map[string]string) (map[st
 	allRemoteMonitorsMap := make(map[string]Monitor)
 	remoteMonitorsSet := mapset.NewSet()
 	// Reversed map for destinations
-	flippedDestinations := utils.ReverseMap(destinationsMap)
+	flippedDestinations := mapIDAsKey(destinationsMap)
```

Comment:
Shouldn't this be `destinations.mapIDAsKey` ?

Reply:
`mapIDAsKey` is a function at the end of this file, should I move it to destinations?

Reply:
Why can't we use existing the `reverseMap` ?  I think we can just reuse that function change argument to interface?

Reply:
`ReverseMap` flips a map of string <-> string. I think it would be more than just changing the signature and the return value to interface rather than string since we're dealing with a specific field 

```
func ReverseMap(m map[string]string) map[string]string {
	n := make(map[string]string)
	for k, v := range m {
		n[v] = k
	}
	return n
}
```